src/main/webapp/app/entities/course.model.ts

```diff
@@ -6,6 +6,11 @@ import { TutorGroup } from 'app/entities/tutor-group.model';
 import { DueDateStat } from 'app/course/dashboards/instructor-course-dashboard/due-date-stat.model';
 import { Exam } from 'app/entities/exam.model';
 
+export const enum Language {
+    ENGLISH = 'ENGLISH',
```

Comment:
this enum already exists somewhere else. please reuse it

Reply:
It is only defined in `tutor-group.model.ts`. Is there a good common/general place I can move this to? (Depending on tutor-group here is kind of weird.)